packages/features/ee/workflows/pages/v2/workflow.tsx

```diff
@@ -189,45 +187,36 @@ function WorkflowPage() {
       <Shell
         title={workflow && workflow.name ? workflow.name : "Untitled"}
         CTA={
-          !isFreeUser && (
-            <div>
-              <Button type="submit">{t("save")}</Button>
-            </div>
-          )
+          <div>
```

Comment:
I believe this div is no longer necessary, if you need a wrapper it can be a fragment instead <></>